packages/devtools_app_shared/lib/src/service/service_extension_manager.dart

```diff
@@ -249,10 +249,14 @@ final class ServiceExtensionManager with DisposerMixin {
       // extension. This will restore extension states on the device after a hot
       // restart. [_enabledServiceExtensions] will be empty on page refresh or
       // initial start.
-      return await _callServiceExtension(
-        name,
-        _enabledServiceExtensions[name]!.value,
-      );
+      try {
+        return await _callServiceExtension(
+          name,
+          _enabledServiceExtensions[name]!.value,
+        );
+      } on SentinelException catch (_) {
+        // Service stopped existing while calling, so do nothing.
```

Comment:
nit r/Service/Service extension. Can you also give an example of when this would happen (i.e. on hot restart?)